### PR TITLE
fix(qgisserver) edit tests are now called

### PR DESCRIPTION
**Pull request by julsbreakdown**
Rename the file so it is called by tests

### CODE REVIEW

File: docker/qgisserver/tests/functional/accesscontrol_allow_to_edit_test.py [hunk 1]
```diff
@@ -19,8 +19,8 @@
 from shapely.geometry import LineString, box, shape
 
 from geomapfish_qgisserver.accesscontrol import Access, GeoMapFishAccessControl, OGCServerAccessControl
-from .accesscontrol_test import add_node_in_qgis_project, set_request_parameters
-
+from .accesscontrol_test import add_node_in_qgis_project, set_request_parameters, \
+    scoped_session, server_iface, qgs_access_control_filter
```
Comment (sbrunner) [inline docker/qgisserver/tests/functional/accesscontrol_allow_to_edit_test.py]: Is that right?
They are never used as python way, just in fixture...

Comment (julsbreakdown) [reply docker/qgisserver/tests/functional/accesscontrol_allow_to_edit_test.py]: Fixtures and their visibility are a bit odd in pytest. They don't require importing, but if you defined them in a test_*.py file, they'll only be available in that file.

Comment (sbrunner) [reply docker/qgisserver/tests/functional/accesscontrol_allow_to_edit_test.py]: Then it's required?

Comment (arnaud-morvan) [reply docker/qgisserver/tests/functional/accesscontrol_allow_to_edit_test.py]: It might sounds good to move them one level up in conftest.py

**Pull request merged**